gdkx11-sys/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Guillaume Gomez
```

Comment:
`gtk-rs developers` we usually put here or not?

Reply:
Copy/pasted from sourceview, so I need to update sourceview as well!